CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Tamr VM Terraform Module
 
+## v3.3.0 - August 4th 2021
+* Adds new variable `emr_abac_valid_tags` to be used in IAM Policies conditions for creating EMR Resources using ABAC
```

Comment:
Does upgrading from v3.2.0 to this go smoothly since it's just an update to the policy?

Reply:
Yes! Terraform definitely won't destroy anything, only updates in-place. Older statements were either kept or merged and we shouldn't expect any problems regarding new permissions as well.